nylas/resources/calendars.py

```diff
@@ -114,25 +114,24 @@ def destroy(self, identifier: str, calendar_id: str) -> DeleteResponse:
         )
 
     def get_availability(
-        self, identifier: str, request_body: GetAvailabilityRequest
+        self, request_body: GetAvailabilityRequest
     ) -> Response[GetAvailabilityResponse]:
         """
         Get availability for a Calendar.
 
         Args:
-            identifier: The grant ID or email account to get availability for.
             request_body: The request body to send to the API.
 
         Returns:
             Response: The availability response from the API.
         """
         json_response = self._http_client._execute(
             method="POST",
-            path=f"/v3/grants/{identifier}/calendar/availability",
+            path=f"/v3/calendar/availability",
             request_body=request_body,
         )
 
-        return Response.from_dict(json_response, GetAvailabilityResponse)
+        return Response(json_response, GetAvailabilityResponse)
```

Comment:
Is this tested to work? The other responses use `from_dict`

Reply:
I removed it because it was generating some weird errors...once removed everything worked fine...test it several times